pip/cmdoptions.py

```diff
@@ -523,6 +524,47 @@ def only_binary():
 )
 
 
+def _good_hashes():
+    """Return names of hashlib algorithms at least as strong as sha256."""
+    # Remove getattr when 2.6 dies.
```

Comment:
https://docs.python.org/3/whatsnew/2.7.html suggests it was added in 2.7.0; it lists [things added in 2.7.x](https://docs.python.org/3/whatsnew/2.7.html#new-features-added-to-python-2-7-maintenance-releases) separately.